static/styles/twodsix.css

```diff
@@ -765,9 +765,18 @@ a.notes-tab.active {
 
 .items-traits {
   display: grid;
-  grid-template-columns: 14em 2em 14em 3em;
+  grid-template-columns: 7em 22em 4em;
```

Comment:
"grid-template-columns: 8em 20em 4em;"

Reply:
If you want just go ahead and push a commit. Otherwise I can fix it.